homeassistant/components/ring/number.py

```diff
@@ -0,0 +1,152 @@
+"""Component providing HA number support for Ring Door Bell/Chimes."""
+
+from collections.abc import Awaitable, Callable
+from dataclasses import dataclass
+from typing import Any, Generic, cast
+
+from ring_doorbell import RingChime, RingDoorBell, RingGeneric, RingOther
+import ring_doorbell.const
+
+from homeassistant.components.number import (
+    NumberEntity,
+    NumberEntityDescription,
+    NumberMode,
+)
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import StateType
+
+from . import RingConfigEntry
+from .coordinator import RingDataCoordinator
+from .entity import RingDeviceT, RingEntity, refresh_after
+
+SKIP_UPDATES_DELAY_SECONDS = 5
+
```

Comment:
```suggestion
```

Reply:
Just did a final check and spotted a variable left behind. Other than that I think this is good to go so approving on the basis that you'll fix this before merge happens.

Reply:
Ah, good catch! I missed it when moving to the new `refresh_after`.